tests/integration/conftest.py

```diff
@@ -18,3 +20,25 @@ def test_env():
     with open(config_file) as f:
         env = yaml.safe_load(f)
     return env
+
+
+@pytest.fixture(scope="session")
+def default_request(test_env):
+    """
+    Create a new request in Cachito.
+
+    :param test_env: Test environment configuration
+    :return: Object that contains initial and completed response from the Cachito API
+    :rtype: tuple
+    """
+    client = utils.Client(test_env["api_url"], test_env["api_auth_type"])
+    initial_response = client.create_new_request(
+        payload={
+            "repo": test_env["package"]["repo"],
+            "ref": test_env["package"]["ref"],
+            "pkg_managers": test_env["package"]["pkg_managers"],
+        },
+    )
+    completed_response = client.wait_for_complete_request(initial_response)
+
+    return initial_response, completed_response
```

Comment:
Sorry again for not mentioning this earlier, but perhaps a named tuple would make this easier to use:
```python
from collections import namedtuple
DefaultRequest = namedtuple('DefaultRequest', 'initial_response complete_response')
```

Then in your tests, you can use it like `default_request.initial_response` and `default_request.complete_response`